### PR TITLE
Change wording in show instances part of HA docs

### PR DESCRIPTION
**Pull request by antoniofilipovic**
### Description

Address changes according to the issue: https://github.com/memgraph/documentation/issues/942

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)
https://github.com/memgraph/documentation/issues/942

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [x] If release-related, add a product and version label
- [x] If release-related, add release note on product PR

### CODE REVIEW

File: pages/clustering/high-availability.mdx [hunk 1]
```diff
@@ -268,15 +268,19 @@ This operation will result in writing to the Raft log.
...
+ 2. Health state of server
+ 3. Role - MAIN, REPLICA, LEADER, FOLLOWER or unknown if not alive
+ 4. The time passed since the last response time to the leader's health ping 
+
+This query can be run on either the leader or followers. Since only the leader knows the exact status of the health state and last response time, 
+followers will execute actions in this exact order:
+  1. Try contacting the leader to get the health state of the cluster, since the leader has all the information. 
+  If the leader responds, the follower will return the result as if the `SHOW INSTANCES` query was run on the leader.
+  2. When the leader doesn't respond or currently there is no leader, the follower will return all the Memgraph servers
+   with the health state and the last response time set to "unknown" and 0, respectively.
```
Comment (hal-eisen-MG) [inline pages/clustering/high-availability.mdx:283]: s/health state/health state set to "down",

**Comment by hal-eisen-MG** [comment]
I'm waiting for the `status: ready` label

**Comment by antoniofilipovic** [comment]
@hal-eisen-MG added label

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**